doc/bql.rst

```diff
@@ -130,17 +130,16 @@ Populations
 +++++++++++
 
 A population specifies which columns in a table should be modeled or ignored.
-For those that are modeled, it specifies whether they should be modeled as
-numerical or nominal.
+For those that are modeled, it specifies their statistical type.
 
 .. index:: ``GUESS SCHEMA``
 
 ``GUESS SCHEMA FOR <name>``
 
    Guess a population schema for the table *name*. A schema maps the columns in
    *name* to heuristically guessed statistical types. Statistical types can be
-   guessed to be NOMINAL or NUMERICAL. Columns can also be guessed to be ignored
-   (IGNORE).
+   guessed to be nominal or numerical. Columns can also be guessed to be
+   ignored.
```

Comment:
I meant the English words here (although I was indecisive about it, which is why I changed them from uppercase to lowercase). Do you prefer them to be noted as BQL tokens? If so, I will change the formatting. I will plan to make a separate document re: statistical types.